apps/backend/src/db/createTable.ts

```diff
@@ -5,23 +5,23 @@ import { qldbDriver, tableName } from './qldb.js';
 // the table in the AWS console so the rest of the application code can assume the
 // table exists.
 export async function createTableIfNotExists() {
-  const tableNames = await qldbDriver.getTableNames();
-  if (!tableNames.includes(tableName)) {
-    await qldbDriver.executeLambda(async (txn) => {
-      await createTable(txn);
-      // await createIndex(txn);
-    });
-  }
+    const tableNames = await qldbDriver.getTableNames();
```

Comment:
looks like your autoformatter does 4 spaces instead of 2, could you change that to 2 in your editor (for consistency's sake) so this PR isn't a bunch of whitespace diffs?